### PR TITLE
CORTX-34395: Ansible script for fetching and installing packages

### PR DESCRIPTION
**Pull request by gauravchaudhari02**
# Problem Statement
- Migration from Cloudform VM's to VMware VM's requires exact package installation from Cloudform to VMware environment VM's.

# Design
-  Written a ansible script which gathers packages from Cloudform VM's and installs it on VMware VM's

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [ ] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] Jira ID/ COMMUNITY is prefixed in PR title
      Prefix Syntax - 
        `CORTX-<5 digit JIRA ID>: <PR Title>` - for Seagate Employees
        `COMMUNITY: <PR Title>`  - for open source contributors
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/vmware/migration/ansible/vm-metadata-operations/roles/install_packages/tasks/install_docker.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  get_url:
+    url: https://download.docker.com/linux/centos/docker-ce.repo
+    dest: /etc/yum.repos.d/docer-ce.repo
+
+- name: Install Docker
+  package:
+    name: docker-ce
+    state: latest
+
+- name: Start Docker service
```
Comment (shailesh-vaidya) [inline solutions/vmware/migration/ansible/vm-metadata-operations/roles/install_packages/tasks/install_docker.yml:17]: @gauravchaudhari02 Can you please add steps to create `/etc/docker/daemon.json`  and stop/start the docker service

**Review by shailesh-vaidya — COMMENTED** [review]
Looks good to me. Just added one minor comment. Please remove commented code.

File: solutions/vmware/migration/ansible/vm-metadata-operations/roles/install_packages/tasks/install_docker.yml (hunk 1 above)
Comment (gauravchaudhari02) [reply solutions/vmware/migration/ansible/vm-metadata-operations/roles/install_packages/tasks/install_docker.yml:17]: @shailesh-vaidya , Added requested tasks

**Review by shailesh-vaidya — APPROVED** [review]

**Review by pankajgborole — APPROVED** [review]
Approved

**Pull request merged**